zipkin/src/main/java/zipkin/storage/QueryRequest.java

```diff
@@ -178,7 +178,7 @@ public static Builder builder() {
     private Long minDuration;
     private Long maxDuration;
     private Long endTs;
-    private Long lookback;
+    private Long lookback = 86400000L; // see ZipkinQueryApiV1.defaultLookback
```

Comment:
this will lead to drift as this default has changed in the past. let's leave it out of this change, pls (maybe just put it in the cassandra tests if you need it)